x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/TransportUnfollowActionTests.java

```diff
@@ -47,6 +49,7 @@ public void testUnfollow() {
         IndexMetaData resultIMD = result.metaData().index("follow_index");
         assertThat(resultIMD.getSettings().get(CcrSettings.CCR_FOLLOWING_INDEX_SETTING.getKey()), nullValue());
         assertThat(resultIMD.getCustomData(Ccr.CCR_CUSTOM_METADATA_KEY), nullValue());
+        assertThat(resultIMD.getSettingsVersion(), equalTo(settingsVersion+ 1));
```

Comment:
Nit on the formatting here (whitespace in front of the + please).

Reply:
Eagle :eyes:, thanks